_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
-| [Support for Mixer in Envoy (Mixer Client Library)](https://github.com/istio/mixerclient)      | Alpha
-
-### Environments
-
-
-| Feature           | Phase        
-|-------------------|-------------------
-| [Kubernetes: Traffic Interception](https://istio.io/docs/setup/kubernetes/)        | Beta
-| [VM: Basic Envoy Installation, Interception and Registration for VMs](https://istio.io/docs/guides/integrating-vms.html) | Alpha
-| [Kubernetes: Istio Installation](https://istio.io/docs/setup/kubernetes/)    | Alpha
-| [Kubernetes: Envoy Installation](https://goo.gl/jhng56)               | Alpha
-| [Kubernetes: Pilot supports K8s Service Discovery](https://istio.io/docs/setup/kubernetes/)         | Alpha
-| [Integration into Consul Service Discovery](https://istio.io/docs/setup/consul/quick-start.html)     		   | Alpha
-| [Bring your own service name/DNS](https://docs.google.com/document/d/1If-t_g7CZnRX_dc0NG4y6CRGoI5FhkgUCqnNmPt9eFY/edit)         	   | Alpha
+| [Prometheus Integration](https://istio.io/docs/guides/telemetry.html)         | Beta
+| [Local Logging (STDIO)](https://istio.io/docs/guides/telemetry.html)               | Beta
+| [Statsd Integration](https://istio.io/docs/reference/config/mixer/adapters/statsd.html)         	   | Beta
+| [Service Dashboard in Grafana](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html)       | Beta
+| [Stackdriver Integration](https://istio.io/docs/reference/config/mixer/adapters/stackdriver.html)       | Alpha
+| [Service Graph](https://istio.io/docs/tasks/telemetry/servicegraph.html)       | Alpha
 
 
 ### Security
 
 
 | Feature           | Phase        
 |-------------------|-------------------
-| [Kubernetes Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)               | Beta
-| [Istio mTLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
+| [Deny Checker](https://istio.io/docs/reference/config/mixer/adapters/denier.html)         | Beta
+| [List Checker](https://istio.io/docs/reference/config/mixer/adapters/list.html)        | Beta
+| [Kubernetes: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)               | Beta
 | [Pluggable Key/Cert Support for Istio CA](https://istio.io/docs/tasks/security/plugin-ca-cert.html)        | Beta
-| [Istio on Istio (secure istio components)](https://docs.google.com/document/d/1YzYPddihbLgJhme27-md9COn0NnKUKX_xCJ_GPXU4Fw/edit#heading=h.jbmfrt4h5lj0) | Alpha
-| [Incremental mTLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
-| [VM Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
+| [Service-to-service mutual TLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
+| [Incremental Enablement of service-to-service mutual TLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
```

Comment:
Can we remove links to docs and code? It is not really helpful in the common case, where people are browsing the web page..

Reply:
Agreed. We have been pushing the teams to write user docs. Jae as an example wrote a readme file for OPA adapter this morning (since I refused to link the design doc)  (https://github.com/mangchiandjjoe/istio/blob/d5390f6e436225949907d77ad3e9747a9bc26722/mixer/adapter/opa/README.md). This one however slipped through from 0.3 table. 
Again, we could really use some help with docs :)

Reply:
We dont need to have docs for each and every thing listed in this page. And once again, this docs thing becomes a non issue when you make the list coarse grained.